CondTools/SiPhase2Tracker/plugins/BuildFile.xml

```diff
@@ -0,0 +1,16 @@
+<use name="CommonTools/UtilAlgos"/>
+<use name="CoralBase"/>
```

Comment:
why this dependency?

Reply:
Mmm... I guess I inherited an "antico vaso". I removed it